README.md

```diff
@@ -1,5 +1,11 @@
 # EJP-RD - NBIS demo setup
 
+**Note**: This repository contains serveral `git submodules` that lock in specific versions of data and configurations located in other repositories. When you have used `git clone` or `git checkout _____` please remember to update the submodules.
+
+```{sh}
```

Comment:
`git submodule init`

is needed the first time (or whenever a new submodule has been added), otherwise `git submodule update` won't do anything